themes/default/content/blog/announcing-6-0-of-the-pulumi-aws-classic-provider/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Announcing 6.0 of the Pulumi AWS Classic Provider"
+date: 2023-08-08T09:00:00-04:00
+draft: false
+meta_desc: "Announcing 6.0 of the Pulumi AWS Classic Provider, including SDK size optimization, TF plugin framework support, new resources, and more."
+meta_image: meta.png
+authors:
+    - mikhail-shilkov
+    - monica-rodriguez
+tags:
+    - aws
+    - providers
+
+---
+
+We are excited to announce 6.0 of the Pulumi AWS Classic provider. The AWS Classic provider is the most heavily used provider across the entire Pulumi ecosystem, and offers access to the full surface area of the upstream Terraform AWS Provider from within Pulumi projects in all supported Pulumi languages. The 6.0 release brings a substantial set of fixes and improvements to the provider, including a number of breaking changes as part of the major version release.
+
+This blog post walks you through the list of notable changes in the new major version.
+
+<!--more-->
+
+{{% notes type="info" %}}
+Join Pulumi on Thursday, September 7, 2023 for a live workshop on [getting started with infrastructure as code on AWS](/resources/getting-started-with-iac-pulumi-aws/), which will feature the new AWS 6.0 provider.
+{{% /notes %}}
+
+### SDK Size Optimization
+
+The AWS Classic provider contains more than 1,200 resources with over 12,000 properties between them. As a result, it’s expected that the SDK size of the provider has grown to be fairly large. With that in mind, we invested in some key areas to reduce the size of generated files. Depending on the runtime, the SDK is now up to 90% smaller than it used to be in v5 of the provider!
+
+![aws_sdk](aws_sdk.png)
+
+One particular optimization is related to `wafv2` resources, that were particularly difficult to use in Pulumi AWS v5 (issue [one](https://github.com/pulumi/pulumi-aws/issues/2276), [two](https://github.com/pulumi/pulumi-aws/issues/1117[), [three](https://github.com/pulumi/pulumi-aws/issues/2250)). The new release includes a refactored `wafv2` module which uses recursive types. Using recursive types allowed us to remove a lot of excess generated code.
+
+Smaller SDK sizes mean faster downloads, snappier experience with editors and code completion tools, less time needed for `pulumi preview` and `pulumi up` operations, and smaller binaries, all of which contribute to a streamlined developer experience for Pulumi users.
+
+### Terraform Plugin Framework Support
```

Comment:
Maybe it’s sort of covered in the next bullet - but feels like we should highlight all of the new resources that we support which are using Plugin Framework?

Also - are there any user-facing benefits we can speak to for Pulumi users?  I know we’ve uplevelled our bridge support significantly here - but not sure there’s any specific benefit we can connect to this?